### PR TITLE
Diego

### PR DESCRIPTION
**Pull request by Dfloresa**
IntInvoice and invoice crud added, lombok anottations also added

### CODE REVIEW

File: IntelliJ Project/src/main/java/com/globant/musicstore/entity/IntInvoiceAlbum.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "int_invoice_album ")
+public class IntInvoiceAlbum {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "int_invoice_album_id ")
+	private Integer int_invoice_album_id ;
```
Comment (quark-works) [inline IntelliJ Project/src/main/java/com/globant/musicstore/entity/IntInvoiceAlbum.java]: Please remember that we don't use snake_case, should be camelCase.

File: IntelliJ Project/src/main/java/com/globant/musicstore/entity/Invoice.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "invoice")
+public class Invoice {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "invoice_id ")
+	private Integer invoice_id ;
```
Comment (quark-works) [inline IntelliJ Project/src/main/java/com/globant/musicstore/entity/Invoice.java]: Here as well, no snake_case.

**Review by quark-works — CHANGES_REQUESTED** [review]

**Comment by quark-works** [comment]
Also, you have a lot of "dot folders", those used for local configuration. Please try and get them out of the commit and only in your local development. 🙏

**Review by JoseVm96 — APPROVED** [review]

**Review by quark-works — APPROVED** [review]
LGTM!

**Review by AdrianaCR17 — APPROVED** [review]

**Review by mike-angel-dev — APPROVED** [review]

**Pull request merged**